__tests__/chat.test.js

```diff
@@ -153,6 +151,30 @@ describe('Chat Methods', () => {
     })
   })
 
+  describe('Chat react', () => {
+    it('Allows a user to react to a valid message', async () => {
+      await alice1.chat.send(channel, message)
+      let messages = await alice1.chat.read(channel, {peek: true})
+      const messageToReactTo = messages[0]
+
+      await bob.chat.react(channel, messageToReactTo.id, ':poop:')
+      messages = await alice1.chat.read(channel, {peek: true})
+      const reaction = messages[0]
+      expect(reaction.id).toBe(messageToReactTo.id + 1)
+      expect(reaction.content.type).toBe('reaction')
+      expect(messages[1].reactions.reactions.poop).toHaveProperty(config.bots.bob1.username)
+    })
+
+    // it('Throws an error if given an invalid emoji', async () => {
+    //   await alice1.chat.send(channel, message)
+    //   const messages = await alice1.chat.read(channel, {peek: true})
+    //   const messageToReactTo = messages[0]
+
+    //   expect(bob.chat.react(channel, messageToReactTo.id, 'blah')).rejects.toThrowError()
+    // })
+    // it("Throws an error if it cannot react to a message (e.g., it's not a reactable message type")
```

Comment:
I couldn't find any message types that I couldn't react too. I could react to delete messages and reaction messages, both of which don't show up in the GUI at all so I'm not sure if this is an intentional feature.

Reply:
not sure either. I would say if it doesn't cause probs don't worry abbout it